public/js/main.js

```diff
@@ -245,4 +270,18 @@ function toggleLoadingState(form) {
 	loader.classList.toggle('cape');
 }
 
+function addLimitsWarning(user = 'user') {
+	var errorMessage = 'The monthly quota for translations has been reached. Please wait until next month or contact #ftlabs for a demo';
```

Comment:
Do we want to say email us or something incase someone using it isnt on slack?

Reply:
Could do. Historically we've referenced Slack, avoids having our team email loose on GitHub as it's different from the public-facing one.